README.md

```diff
@@ -1,2 +1,135 @@
+Automatically upload project files to the server* is activated. Do not worry about the rest for now and click finish.
+
+![Configure Interpreter 2](img/configure_interpreter_2.png)
+
+Step 7: Go to `File` &rarr; `Settings` &rarr; `Build, Execution & Deployment` &rarr; `Deployment`. The created ssh
+connection should now show up with type *SFTP*. Click on `AUTODETECT` in the connection tab. You should see `/home/ros`.
+
+![Deployment Settings](img/deployment_settings.png)
+
+Step 8: In the mappings tab, change the local path, deployment path, and web path as shown in the picture below. This
+steps make sure that the syncing of files between the host and server is correct. Click apply, and finally make sure
+that the new connection is used as default (`ros@localhost:3456` in the list of connections should be bold, otherwise
+use the tick above to set it as default).
+
+![Deployment Settings 2](img/deployment_settings_2.png)
+
+Step 9: Finally, go to `File` &rarr; `Settings` &rarr; `Build, Execution & Deployment` &rarr; `Deployment`
+&rarr; `Options` and change to `Yes` under *Preserve original file permissions (SFTP only)*. Apply and close.
+
+![SFTP Settings](img/sftp_settings.png)
+
+Step 10 (for testing): Right click on your project folder, go to *Deployment* and click on *Upload to ros@localhost:
+3456*. You should get a message in green saying that the files have been transferred.
+
+Step 11 (optional): This example can easily be extended to a project with several ROS packages. Assuming a folder
+structure like
+
+```bash
+ros-pycharm-example
+   └──source
+       ├──package1
+       └──package2
+```
+
+you would just need to `Add new mapping` in step 8 such that you have one mapping per package. Additionally, after you
+have built the package(s) the first time, you can go to `File` &rarr; `Settings` &rarr; `Project: ros-pycharm-example`
+&rarr; `Python Interpreter` and choose *Show All*.
+
+![Add dist packages](img/add_dist_packages.png)
+
+Then, select your new interpreter and click on *Show paths for selected interpreter* and then click on the plus.
+
+![Add dist packages 2](img/add_dist_packages_2.png)
+
+Finally, choose to add `/home/ros/ros_ws/devel/lib/python3/dist-packages` as additional remote path. This will enable
+full IDE capabilities even for the package(s) currently under development in the project.
+
+![Add dist packages 3](img/add_dist_packages_3.png)
+
+## Using the example
+
+Once the server is running in the background, you can use
+the [`connect.sh` script](https://github.com/aica-technology/docker-images/blob/master/scripts/connect.sh)
+to attach new terminal windows by specifying the container name and user. You can use `docker container ls` to lookup
+the names of running containers. In the case of this example:
+
+```shell script
+/.../aica-technology/docker-images/scripts/connect.sh domire8-ros-pycharm-example-ssh --user ros
```

Comment:
just for consistency, it would be nice if this was also 
```suggestion
path/to/aica-technology/docker-images/scripts/connect.sh domire8-ros-pycharm-example-ssh --user ros
```
(or conversely, if the server.sh one was the same as this one)